js/dialogcards.js

```diff
@@ -476,11 +661,50 @@ H5P.Dialogcards = (function ($, Audio, JoubelUI) {
       self.stopAudio(self.$current.index());
       self.$current.removeClass('h5p-dialogcards-current');
       self.$current = $prev.addClass('h5p-dialogcards-current').removeClass('h5p-dialogcards-previous');
+      self.turnCardToFront ();
       self.updateNavigation();
     }
   };
 
   /**
+   * User selected cards order option (normal/random).
+   */
+  C.prototype.randomizeOrder = function (randomizeYesNo) {
```

Comment:
Strange parameter name. Could it have been a boolean? (Just quickly reading through the code, so I don't know what values it may take)